plugins/packs-integrations.js

```diff
@@ -356,7 +356,7 @@ async function getLogoUrl(packsAllData, logoUrlMap) {
             ? `${BASE_URL}${url}`
             : `${BASE_URL}/${url}`;
 
-      if (!logoUrlMap.hasOwnProperty(packName)) {
+      if (!Object.prototype.hasOwnProperty.call(logoUrlMap, packName)) {
```

Comment:
This seems like it came from your other PR?

Reply:
@addetz I did this to appease eslint. Otherwise we get this error

```
 359:23  error  Do not access Object.prototype method 'hasOwnProperty' from target object  no-prototype-builtins
 ```